src/include/catalog/settings_catalog.h

```diff
@@ -45,7 +47,7 @@ class SettingsCatalog : public AbstractCatalog {
   std::string GetDefaultValue(const std::string &name,
                               concurrency::TransactionContext *txn);
 
-  enum class ColumnId {
+  enum ColumnId {
```

Comment:
It's probably a good idea to stick to strongly typed enums instead of the old-style c++ enums. 
Old-style enums don't have their own scope, are not very portable and have a bunch of other issues.

https://www.ibm.com/developerworks/rational/library/scoped-enums/index.html -  for a more detailed read.

The same below.

Reply:
Agreed.

@Onmysofa why did you guys remove the `class' modifier?